test/optimizations/constDomain/domainArgs.chpl

```diff
@@ -0,0 +1,83 @@
+const colWidth = 12;
+const fmt = "%-"+colWidth:string+"s|%-"+colWidth:string+"s\n";
+
+//// this is const ref -- `const` doesn't matter here
+proc defIntent(d) {
+  writef(fmt, "(const ref)", d.definedConst);
+}
+
+// TODO: Something's wrong with generic in intents. File a bug report, and
+// make this generic once it is resolved
+//
+// It causes the copy to be dropped for a domain literal argument (maybe
+// correctly?), so this reports 'true' for a domain literal. For concrete
+// argument, the copy remains there and we get 'false' (correctly).
```

Comment:
Is this comment still accurate? If so, note future work in the PR message and make a .future and issue for it.

Reply:
This is actually captured in https://github.com/chapel-lang/chapel/issues/16398, but I'll also modify the PR message and will also replace that TODO with the link to the issue